taichi/backends/wasm/aot_module_builder_impl.h

```diff
@@ -24,7 +24,8 @@ class AotModuleBuilderImpl : public AotModuleBuilder {
   void add_per_backend_tmpl(const std::string &identifier,
                             const std::string &key,
                             Kernel *kernel) override;
-  void add_per_backend_field(const std::string &Identifier,
+  void add_field_per_backend(const std::string &Identifier,
```

Comment:
When we have the snode, is the identifier still important?

Reply:
It kind of is. Users will be able to retrieve the info of the field (represented by this SNode) via the identifier.